configurations/manager/10GB_dataset.yaml

```diff
@@ -0,0 +1,7 @@
+round_robin: true
```

Comment:
If there are no plans to reuse this 10GB dataset in any other test with these settings, better move it to `test-cases/manager/manager-no-delta-backup-set-dataset.yaml`

Reply:
I mean whole file, not only `round_robin` setting.

Reply:
I'd prefer to keep it here in separate file.
It allows us to be more flexible and reuse this dataset in another tests or run this test with different dataset.
It's sometimes a case for the runs which we trigger manually from time to time.

Reply:
okay, I got the point, the test should be fully functioning with only test-cases configuration provided, will move

Reply:
Done